content/pages/sprint/2022/12-hsinchu.rst

```diff
@@ -0,0 +1,77 @@
+========================================
+Scisprint 2022 December in Hsinchu
+========================================
+
+:date: 2022-11-24 10:00
+:url: sprint/2022/12-hsinchu
+:save_as: sprint/2022/12-hsinchu.html
+
+Scisprint 2022 December in Hsinchu
+========================================
+
+* Date: 3rd December, Saturday, 2022
+* Time: 13:00 -- 17:00 (4 hours)
+
+To join the sprint, please bring your laptop and sign up in the `sprint note
+<#sprint-note>`__.  You are also very welcome to bring your project, which
+should be added in the `sprint note <#sprint-note>`__ too.  Please `contact us
+<#contact-us>`__ if you have any questions.
+
+Scisprint is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+In this sprint, we provide a `Sprint Training room <#Sprint Training room>`__ 
```

Comment:
Could you double check whether or not the target works?  IIUC rst connects the words using hyphen (`-`).

Please also correct the capitalization.  If the first two words are capitalized, the third one should be as well.